presto-kafka/src/test/java/io/prestosql/plugin/kafka/encoder/json/TestSecondsJsonDateTimeFormatter.java

```diff
@@ -57,5 +78,12 @@ public void testSecondsDateTimeFunctions()
         testTimestamp(sqlTimestampOf(3, 2020, 8, 18, 12, 38, 29, 0), LocalDateTime.of(2020, 8, 18, 12, 38, 29, 0).toEpochSecond(ZoneOffset.UTC));
         testTimestamp(sqlTimestampOf(3, 1970, 1, 1, 0, 0, 0, 0), 0);
         testTimestamp(sqlTimestampOf(3, 1800, 8, 18, 12, 38, 29, 0), LocalDateTime.of(1800, 8, 18, 12, 38, 29, 0).toEpochSecond(ZoneOffset.UTC));
+
+        testTimestampWithTZ(sqlTimestampWithTimeZoneOf(3, 2020, 8, 18, 12, 38, 29, 123000000, UTC_KEY), getEpochSecondsFromDateTimeWithZone(2020, 8, 18, 12, 38, 29, 123000000, UTC_KEY));
```

Comment:
that's very repetitive, but maybe that's fine... let's format this on lines:

```
        testTimestampWithTZ(
                sqlTimestampWithTimeZoneOf(3, 2020, 8, 18, 12, 38, 29, 123000000, UTC_KEY),
                ZonedDateTime.of(2020, 8, 18, 12, 38, 29, 123000000, UTC).toEpochSecond());
```

and remove `getEpochSecondsFromDateTimeWithZone` method